### ISSUE TITLE
`no-get` incorrectly warns on properties with unconventional characters

### ISSUE THREAD

**Issue by alexlafroscia** [ember-cli/eslint-plugin-ember#609]
I noticed after running the fixer for `ember/no-get` that it catches some cases that it really shouldn't.

For example, if the property has a character like a hyphen, which is not actually valid in an identifier, it currently warns but probably shouldn't.

This pattern

```
get(this, 'my-id');
```

was "fixed" into this

```
this.my-id
// which is really
this.my - id
```

Which is not the same thing!
